src/main/java/graphql/language/AstValueHelper.java

```diff
@@ -27,6 +28,8 @@
 import java.util.List;
 import java.util.Map;
 
+import static graphql.schema.GraphQLTypeUtil.*;
```

Comment:
I think your formatter settings is not correct :-) 
See: https://github.com/graphql-java/graphql-java/blob/master/graphql-java-code-style.xml

Reply:
will fix